dnf/repo.py

```diff
@@ -886,7 +886,9 @@ def load(self):
 
         """
         if self.metadata or self._try_cache():
-            if self._check_config_file_age and self.repofile \
+            if self.metadata_expire != -1 \
```

Comment:
@ignatenkobrain I will update documentation to: "metadata_expire: ... Set this to less than 0 or never to make the repo never considered expired.". Also `never` gets transformed to `-1`, when the option is being read from config file.